aspnetcore/migration/50-to-60.md

```diff
@@ -146,17 +146,31 @@ The preceding code migrated to the new minimal hosting model:
 
 [!code-csharp[](~/migration/50-to-60/samples/WebRP31to60/Startup.cs?name=snippet_60)]
 
+In the preceding code, the ` if (env.IsDevelopment())` block is removed because in [development mode](xref:fundamentals/environments), the developer exception page middleware is enabled by default. For more information, see [Differences between the ASP.NET Core 5 and 6 hosting models](#diff) in the next section.
+
 When using a custom dependency injection (DI) container, add the following highlighted code:
 
 [!code-csharp[](~/migration/50-to-60/samples/WebRP31to60/Program.cs?name=snippet_60a&highlight=11-13)]
 
 [!code-csharp[](~/migration/50-to-60/samples/WebRP31to60/Startup.cs?name=snippet_60a&highlight=17-21)]
 
+<a name="wraps"></a>
+
+When using the minimal hosting model, the endpoint routing middleware wraps the entire middleware pipeline, therefore there's no need to have explicit calls to `UseEndpoints` to register routes. `UseRouting` can still be used to specify where route matching happens but `UseRouting` doesn't need to be explicitly called.
```

Comment:
Nit:

```suggestion
When using the minimal hosting model, the endpoint routing middleware wraps the entire middleware pipeline, therefore there's no need to have explicit calls to `UseRouting` or `UseEndpoints` to register routes. `UseRouting` can still be used to specify where route matching happens, but `UseRouting` doesn't need to be explicitly called if routes should be matched at the beginning of the middleware pipeline.
```

I think this makes it more clear where route matching happens by default. This is a little more wordy and possibly redundant given this already indicates " endpoint routing middleware wraps the entire middleware pipeline", so I understand if you don't take this suggestion.